README.md

```diff
@@ -22,13 +22,20 @@ We recommend using the HACS installation method, which makes future updates to K
 
 ## Configuration
 
-Configuration is now possible using the GUI:
+Configuration is possible using the Home Assistant user interface, or using YAML. We recommend the UI method.
```

Comment:
I'm not certain YAML configuration still works. I think it might have been stripped out. 

My online presence will be limited for the next several days. Maybe you or another contributor can check.

Reply:
Yeah... YAML configuration no longer works. The integration gets loaded but the indoor units are not discovered. I think some code for that was removed in the recent major version.

Please remove the section describing YAML configuration.

Reply:
@dlarrick Sorry for the delay, couldn't get to testing it over the weekend. I've removed the YML config details. Probably could reformat the options more, but for now, just wanted to get this in.